Framework/DataHandling/src/LoadMask.cpp

```diff
@@ -300,24 +519,16 @@ void LoadMask::bankToDetectors(const std::vector<std::string> &singlebanks,
     detid_t detid_last = idetectors.back()->getID();
 
     // b) set detectors
-    if (detid_first + int32_t(numdets) == detid_last + 1 && false) {
-      // TODO This save-time method is not used at this stage
-      g_log.information() << "Using Range of Detectors\n";
-
-      detectorpairslow.push_back(detid_first);
-      detectorpairsup.push_back(detid_last);
-
-    } else {
-      g_log.debug() << "Apply 1 by 1  "
-                    << "DetID: " << detid_first << ", " << detid_last << '\n';
 
-      for (const auto &det : idetectors) {
-        int32_t detid = det->getID();
-        detectors.push_back(detid);
-      }
+    for (const auto &det : idetectors) {
+      int32_t detid = det->getID();
+      detectors.push_back(detid);
```

Comment:
Not that it matters here, but I guess one should generally prefer `emplace_back` over `push_back`.